packages/react-native-sdk/src/theme/theme.ts

```diff
@@ -585,4 +610,32 @@ export const defaultTheme: Theme = {
     buttonIcon: {},
     buttonText: {},
   },
+  getValue: function (component: string, prop: string) {
+    if (this[component] && this[component][prop]) {
+      return this[component][prop];
+    }
+
+    switch (prop) {
+      case 'color':
+        return this.defaults.color;
+      case 'backgroundColor':
+        return this.defaults.backgroundColor;
+      case 'margin':
+        return this.defaults.margin;
+      case 'padding':
+        return this.defaults.padding;
+      case 'fontSize':
+        return this.defaults.fontSize;
+      case 'fontWeight':
+        return this.defaults.fontWeight;
+      case 'borderRadius':
+        return this.defaults.borderRadius;
+      case 'borderColor':
+        return this.defaults.borderColor;
+      case 'borderWidth':
+        return this.defaults.borderWidth;
+      default:
+        throw new Error(`Invalid component or prop: ${component}.${prop}`);
```

Comment:
I'm not sure if we should throw here. Missing theme property shouldn't crash the application. I think we should just log an error.

Also, as this is a forever-growing list, I think we can implement it like this:
```ts
if (this[component] && this[component][prop]) {
  return this[component][prop];
}

const defaultValue = this.defaults[prop];
if (!defaultValue) {
  // log
}
return defaultValue;
```